data/base/script/campaign/cam2-a.js

```diff
@@ -209,7 +209,10 @@ function cam2Setup()
 	camCompleteRequiredResearch(COLLECTIVE_RES, THE_COLLECTIVE);
 	camCompleteRequiredResearch(ALPHA_RESEARCH_NEW, CAM_HUMAN_PLAYER);
 
-	camUpgradeOnMapTemplates(cTempl.commc, cTempl.commrp, THE_COLLECTIVE);
+	if (difficulty >= HARD)
```

Comment:
I don't think we should have Medium Cannon tanks on the map with Normal difficulty. I prefer `if (difficulty >= MEDIUM)` as you did in the following levels.